p2p/src/config.rs

```diff
@@ -92,6 +93,9 @@ pub struct P2pConfig {
     pub ping_check_period: PingCheckPeriod,
     /// When a peer is detected as dead and disconnected
     pub ping_timeout: PingTimeout,
+    /// Maximum acceptable time difference between this node and the remote peer.
+    /// If a large difference is detected, the peer will be disconnected.
+    pub max_clock_diff: MaxClockDiff,
```

Comment:
Like the other PR comments, add "in seconds"

Reply:
I don't think it's needed here because `MaxClockDiff` is stored internally as `Duration`.